doc/supported-hardware.md

```diff
@@ -0,0 +1,81 @@
+> in following format: `<nic_name>: <vender_id> <pf_device_id> <vf_device_id>`.
+> Then restart the config daemon and operator webhook pods.
+
+## Supported features per hardware
+
+The following table depicts the supported SR-IOV hardware features of each supported hardware
+
+| Model                    | SR-IOV Kernel | SR-IOV DPDK | SR-IOV Hardware Offload (switchdev) |
+| ------------------------ | ------------- | ----------- |------------------------------------ |
+| Intel XXV710 Family | V | V | X |
+| Intel E810-CQDA2/2CQDA2 Family | V | V | X |
+| Intel E810-XXVDA4 Family | V | V | X |
+| Intel E810-XXVDA2 Family | V | V | X |
+| Mellanox MT27700 Family [ConnectX-4] | V | V | V |
+| Mellanox MT27710 Family [ConnectX-4 Lx] | V | V | V |
+| Mellanox MT27800 Family [ConnectX-5] | V | V | V |
+| Mellanox MT28800 Family [ConnectX-5 Ex] | V | V | V |
+| Mellanox MT28908 Family [ConnectX-6] | V | V | V |
+| Mellanox MT28908 Family [ConnectX-6 Dx] | V | V | V |
+| Mellanox MT42822 BlueField-2 integrated ConnectX-6 Dx | V | V | V |
+| Qlogic QL45000 Series 50GbE Controller | V | X | X |
+
+# Adding new Hardware
+
+## Initial support
+Vendors (or other parties) interested in adding a new supported hardware to the supported list of devices of sriov-network-operator
+should follow the following procedure:
+
+* Open an Issue requesting to add support for new hardware
+  * Specify which hardware it is (Model, Vendor ID, Device ID)
+* Perform Testing on requested hardware using:
+  * Kubernetes last release
+  * Either last release version or master version of sriov-network-operator
+  * Note: We do not have a specifc list of test-cases however you should at a minimum ensure
+      sriov-network-operator is able to properly discover your device, configure SR-IOV and
+      expose them as a kubernetes node resource, and you are able to run workloads consuming
+      those resources.
+* Add information of what was tested to the issue opened
+* Add contact point information to [vendor-support.md](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/doc/vendor-support.md), so we know who to reach out if issues arise when running sriov-network-operator against the specified hardware.
+* Submit PR to add your device to this file as well as to supported-nic-ids configMap [here](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deployment/sriov-network-operator/templates/configmap.yaml) and [here](https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/master/deploy/configmap.yaml).
+  * The tables above should be updated according to what was tested 
+
+## Continuous support
+To ensure sriov-network-operator continues to operate as expected on supported hardware it is recommended that hardware vendors (or another party)
+adds CI which runs against PRs in the project. Without it we cannot commit for sriov-network-operator to continue to work properly on the specified
+hardware.
+
```

Comment:
What happens when a vendor stops support? Do we then remove the device from the list?

Reply:
see note at the end. do you think we need to add more info ?

Reply:
It's actually just a question from my side. What happens if they don't follow through: i.e. they stop the CI, what happesn in that case?

Reply:
if vendor stops supporting CI, then we are back and initial support stage, if we get issues on the specific hardware and no response from contact point, We will raise it the weekly meetings, vote and remove support.

Reply:
ok, clear. Thanks